src/data/code-sample-definitions/acs-users.yaml

```
@@ -0,0 +1,105 @@
+  response:
+    body:
+      acs_user:
+        acs_user_id: 123e4567-e89b-12d3-a456-426614174000
+        acs_system_id: 123e4567-e89b-12d3-a456-426614174000
+        acs_access_group_ids: [123e4567-e89b-12d3-a456-426614174000]
+        workspace_id: 123e4567-e89b-12d3-a456-426614174000
+        created_at: 2024-04-05T07:57:05.323Z
+        display_name: Jane Doe
+        full_name: Jane Doe
+        email_address: jane@example.com
+        phone_number: +15555550100
+- title: Update an ACS User
+  description: Update the properties of a specific [ACS user](https://docs.seam.co/latest/capability-guides/access-systems/user-management).
+  request:
+    path: /acs/users/update
+    parameters:
+      acs_user_id: 8d7e0b3a-b889-49a7-9164-4b71a0506a33
+      phone_number: +12222222222
+  response:
+    body:
+- title: Suspend an ACS User
+  description: "[Suspends](https://docs.seam.co/latest/capability-guides/access-systems/user-management/suspending-and-unsuspending-users#suspend-an-acs-user) a specified [ACS user](https://docs.seam.co/latest/capability-guides/access-systems/user-management). Suspending an ACS user revokes their access temporarily. To restore an ACS user's access, you can [unsuspend](https://docs.seam.co/latest/api/acs/users/unsuspend) them."
+  request:
+    path: /acs/users/suspend
+    parameters:
+      acs_user_id: 8d7e0b3a-b889-49a7-9164-4b71a0506a33
+  response:
+    body:
+- title: Unsuspend an ACS User
+  description: '[Unsuspends](https://docs.seam.co/latest/capability-guides/access-systems/user-management/suspending-and-unsuspending-users#unsuspend-an-acs-user) a specified suspended [ACS user](https://docs.seam.co/latest/capability-guides/access-systems/user-management). While [suspending an ACS user](https://docs.seam.co/latest/api/acs/users/suspend) revokes their access temporarily, unsuspending the ACS user restores their access.'
+  request:
+    path: /acs/users/unsuspend
+    parameters:
+      acs_user_id: 8d7e0b3a-b889-49a7-9164-4b71a0506a33
+  response:
+    body:
+- title: Delete an ACS User
+  description: "Deletes a specified [ACS user](https://docs.seam.co/latest/capability-guides/access-systems/user-management) and invalidates the ACS user's [credentials](https://docs.seam.co/latest/capability-guides/access-systems/managing-credentials)."
+  request:
+    path: /acs/users/delete
+    parameters:
+      acs_user_id: 8d7e0b3a-b889-49a7-9164-4b71a0506a33
+  response:
+    body:
+- title: Add an ACS User to an Access Group
+  description: 'Adds a specified [ACS user](https://docs.seam.co/latest/capability-guides/access-systems/user-management) to a specified [access group](https://docs.seam.co/latest/capability-guides/access-systems/assigning-users-to-access-groups).'
+  request:
+    path: /acs/users/add_to_access_group
+    parameters:
+      acs_user_id: 8d7e0b3a-b889-49a7-9164-4b71a0506a33
+      acs_access_group_id: 123e4567-e89b-12d3-a456-426614174000
+  response:
+    body:
```

Try this instead to fix the build
```suggestion
    body: null
```